webapp/pages/settings/blocked-users.vue

```diff
@@ -92,12 +96,21 @@ export default {
         avatar: '',
         name: this.$t('settings.blocked-users.columns.name'),
         slug: this.$t('settings.blocked-users.columns.slug'),
+        unblock: this.$t('settings.blocked-users.columns.unblock'),
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-11-04T07:23:46Z" title="Monday, November 4th 2019, 8:23:46 am +01:00">Nov 4, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -92,12 +96,20 @@ export default {
         avatar: '',
         name: this.$t('settings.blocked-users.columns.name'),
         slug: this.$t('settings.blocked-users.columns.slug'),
+        unblock: this.$t('settings.blocked-users.columns.unblock'),
```


Ah, this is exactly the mistake I wanted to avoid with my suggestion in the first comment! :grin: I think there is no `blocked-users.columns.unblock` there is just `blocked-users.columns.unblk` or `blocked-users.unblock`.

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-11-05T13:10:19Z" title="Tuesday, November 5th 2019, 2:10:19 pm +01:00">Nov 5, 2019</time>_

---


ok